devops/openshift/api-deploy.yml

```diff
@@ -322,14 +325,14 @@ parameters:
     value: "2"
   - name: PORT
     required: true
-  - name: DB_SERVICE
-    value: patroni-master
+  - name: DB_SERVICE_KEY
+    value: pgbouncer-host
   - name: DB_SECRET_NAME
-    value: patroni-creds
+    value: simsdb-pguser-app-database-user
   - name: DB_USERNAME_KEY
-    value: database-user
+    value: user
```

Comment:
The API and DB Migrations are are using the same user here. I believe the idea for apps(api, workers etc..) is to use non privileged user right?? 

Or is this part going to be handled later?

Reply:
We have different user, if you check the SECRET_NAME_KEY both are pointing to 2 different secrets, but the DB_USERNAME_KEY looks the same 
<img width="557" alt="image" src="https://github.com/user-attachments/assets/5124df75-0d3d-4525-8367-e5f9ceaee524">
<img width="682" alt="image" src="https://github.com/user-attachments/assets/2c7ccaf8-2eaa-4e81-9082-b0352db3a230">